README.md

```diff
@@ -16,26 +19,29 @@ This component provides a collection of icons for use in your **React** web and
 
 **rocketicons** was created to provide a simple and
 easy-to-use icon library for developers who wish to write a single code that can
-work both in **React** web and **React Native**.
+work both in **React** web and **React Native**. We got inspired by the awesome work done by [@kamijin_fanta](https://github.com/kamijin-fanta) on [react-icons](https://github.com/react-icons/react-icons) and decided to improve on top of that.
 
-## Technical specifications
+We also got excited about [@expo](https://github.com/expo) and the possibility of creating multiplatform apps using a single codebase, sharing as much code as we can between multiple platforms.
 
-- Next.js 14
+## Packages in this repository
 
-## Preparing the environment
+- [core](https://github.com/rocketclimb/rocketicons/blob/main/packages/core) - The core of the icon package generation.
+- [gen](https://github.com/rocketclimb/rocketicons/blob/main/packages/gen) - The icon generator itself.
```

Comment:
I think we should rename gen. Gen it the name the original repository used, maybe change this name to generator to disassociate from the original work can be better...

Reply:
Agree